lib/src/constants/constants.dart

```diff
@@ -1,6 +1,9 @@
 abstract class Constants{
-  static const double height = 190.0;
-  static const double width = 300.0;
+  static const double principalContainerheight = 190.0;
+  static const double principalContainerwidth = 300.0;
   static const double padding = 25.0;
-  static const double fontSize = 35.0;
+  static const double textFontSize = 35.0;
+  static const double shadowBlurRadius = 1.0;
+  static const double shadowOffsetX = 3.0;
+  static const double shadowOffsetY = 2.0;
 }
```

Comment:
you are missing an extra line here, ctrl + alt + L should fix it

Reply:
Fixed!